src/app/backend/handler/apihandler.go

```diff
@@ -816,13 +850,14 @@ func (apiHandler *APIHandler) handleDeploy(request *restful.Request, response *r
 }
 
 // Handles deploy from file API call.
-func (apiHandler *APIHandler) handleDeployFromFile(request *restful.Request, response *restful.Response) {
+func (apiHandler *APIHandler) handleDeployFromFile(apiClient *clientK8s.Clientset, request *restful.Request, response *restful.Response) {
 	deploymentSpec := new(deployment.AppDeploymentFromFileSpec)
 	if err := request.ReadEntity(deploymentSpec); err != nil {
 		handleInternalError(response, err)
 		return
 	}
 
+	// TODO inject bearer token
```

Comment:
deployment.DeployAppFromFile takes apiHandler.clientConfig directly. From my perspective I have 2 options:
- Modify deployment.DeployAppFromFile to take an apiClientSet (I guess there was a good reason to use clientConfig directly)
- Figure out a way to inject the token into the clientConfig directly.